.github/matrix.json

```
@@ -1,5 +1,5 @@
 {
   "postgresql": ["12"],
   "ruby": ["2.7", "3.0"],
-  "node": ["14"]
+  "node": ["18"]
```

This should be both 14 and 18 now:
```suggestion
  "node": ["14", "18"]
```